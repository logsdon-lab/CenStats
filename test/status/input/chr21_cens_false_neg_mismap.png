version https://git-lfs.github.com/spec/v1
oid sha256:1bee2866e476995b1cbc26cc25cb52633ae37417e70471a1e179ecdca3693e75
size 103974
