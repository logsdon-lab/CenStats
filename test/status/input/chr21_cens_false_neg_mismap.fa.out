version https://git-lfs.github.com/spec/v1
oid sha256:dad4ede7db7c31ec54cf05bb74884be17756b3edf2076a3e08a973a28e3841cc
size 111924
